docs/contributing-to-airbyte/README.md

```diff
@@ -24,6 +24,35 @@ Here is a list of easy [good first issues](https://github.com/airbytehq/airbyte/
 
 ## Areas for contributing
 
+We gladly welcome all improvements existing on the codebase. 
+
+#### 1. Open an issue, or find a similar one.
+Before jumping into the code, we ask you to create an issue on [GitHub](https://github.com/airbytehq/airbyte/issues/new/choose) to explain what you want to achieve.
+It will enable our team to make sure your contributions do not overlap with existing works and will comply with the design orientation we are currently heading the product toward.
+If you don't receive an update from our team, please ping us on [Slack](https://slack.airbyte.io)!
+And finally, before opening the issue, please check first if [similar one](https://github.com/airbytehq/airbyte/issues) is already open (please filter with the *area/connectors* or *area/platform* labels).
```

Comment:
I would move the part about first checking for issues at the very beginning before creating an issue

Reply:
Makes sense 👍